tests/projects/test_routes_canvas.py

```
@@ -0,0 +1,188 @@
+    c, p, app = client
+
+    # Publish a canvas event into the broker replay buffer first
+    # so the SSE subscriber gets it immediately on subscribe.
+    from tinyagentos.projects.events import ProjectEvent
+    await app.state.project_event_broker.publish(
+        p["id"],
+        ProjectEvent(kind="canvas.element_added", payload={"id": "e1"}),
+    )
+
+    lines = await _collect_canvas_sse(app, p["id"], n_lines=1, timeout=3.0)
+    data_lines = [l for l in lines if l.startswith("data:")]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)